docs/source/advanced/backends.rst

```diff
@@ -16,24 +16,25 @@ Users had to change their programs to use other packages with different API.
 Since 1.6, PyVISA is a frontend to VISA. It provides a nice, Pythonic API and
 can connect to multiple backends. Each backend exposes a class derived from
 VisaLibraryBase that implements the low-level communication. The ctypes wrapper
-around NI-VISA is the default backend (called **ni**) and is bundled with
-PyVISA for simplicity.
+around IVI-VISA is the default backend (called **ivi**) and is bundled with
+PyVISA for simplicity. In general, IVI-VISA can be NI-VISA, R&SVISA, tekVISA etc.
```

Comment:
In general I would like to see Keysight also cited. I use their implementation often at work.